.github/workflows/release.yml

```diff
@@ -19,7 +19,8 @@ jobs:
           python-version: "3.10"
 
       - name: Install Poetry
-        run: python install-poetry.py -y
+        run: |
+          curl -sL https://install.python-poetry.org | python - -y
```

Comment:
In poetry-core we use `curl -sSL`. Makes sense, doesn't it?

Reply:
Sure, this was missing in the main workflow as well, so added it there too.